Fixed bugs in lesson-2, which were pointed out by the mentor.

petrovsky747 opened the pull request:
Lessons are separated by a horizontal line.
Added directory "images" for storing images.
Added new metadata and objects to index.html.
New objects: img, lists, table, form.
Form includes the following objects: input like "name", "email", "password", "range", "checkbox", "radio", textarea, select, button.

petrovsky747 committed:
Fixed bugs in lesson-2, which were pointed out by the mentor.
Lessons are separated by a horizontal line.
Added directory "images" for storing images.
Added new metadata and objects to index.html.
New objects: img, lists, table, form.
Form includes the following objects: input like "name", "email", "password", "range", "checkbox", "radio", textarea, select, button.

5tronciy commented on index.html:
```diff
@@ -2,12 +2,172 @@
+              type="range"
+              name="level"
+              value="5"
+              min="0"
```
0 doesn't match the range 1-10

5tronciy commented on index.html:
```diff
@@ -2,12 +2,172 @@
+               type="email"
+               name="email"
+               placeholder="Your email"
+        />
```
It is rather strange padding.
You may use 1 tab for every attribute.
```
<input
  required
  id="email"
  ...
/>
```
Or leave it in one row like row 106.

5tronciy commented on index.html:
```diff
@@ -2,12 +2,172 @@
+                name="comment"
+                placeholder="Your comment"
+                maxlength="500">Comment
+          </textarea>
```
Close tag should be on the same level as open tag.

5tronciy reviewed (approved):
Looks good.
Not merged just to allow you fixing the remarks. Don't leave it to the next PR.

petrovsky747 committed:
Fixed padding in the following objects:
- input of type "email", "password", "range"
- textarea .
Fixed minimum value of input type "range" from 0 to 1.
